.github/workflows/tests.yml

```diff
@@ -397,9 +400,11 @@ jobs:
         echo ""
 
         echo "----- Install sccache -----"
-        brew update
-        brew install sccache
-        mkdir -p /Users/runner/Library/Caches/Mozilla.sccache
+        curl -L https://github.com/mozilla/sccache/releases/download/v0.5.4/sccache-v0.5.4-x86_64-apple-darwin.tar.gz | tar xz
+        mv -f sccache-v0.5.4-x86_64-apple-darwin/sccache /usr/local/bin
```

Comment:
I would like it if we could avoid this becoming stale. I don't see an easy fix for this though, so @BradyBonnette, do you mind subscribing to the github notifications for new sccache releases and keeping this up to date?

(I would volunteer but rust project stuff overwhelms my github notification box, and I'd definitely fail to notice a release)

Reply:
There might be something we could do internally. Ill take this conversation into Discord because there are considerations here.